rector.php

```diff
@@ -104,6 +104,9 @@
         rex_package_manager::class => rex_addon_manager::class,
         rex_dir::class => Redaxo\Core\Filesystem\Dir::class,
         rex_file::class => Redaxo\Core\Filesystem\File::class,
+        rex_form_base::class => Redaxo\Core\Form\AbstractForm::class,
+        rex_form::class => Redaxo\Core\Form\Form::class,
```

Comment:
Weiter unten in der rector.php werden rex_form und rex_form_base nochmal verwendet. Da dann auf den neuen Namen anpassen.